packages/ai-jsx/src/core/completion.tsx

```diff
@@ -55,17 +55,19 @@ export interface FunctionParameter {
  * This is internal and users should not need to access this directly.
  */
 function AutomaticCompletionModel({ children, ...props }: ModelPropsWithChildren) {
-  if (process.env.OPENAI_API_KEY) {
+  if (process.env.OPENAI_API_KEY || process.env.OPENAI_API_BASE_URL) {
```

Comment:
consider using `OPENAI_API_BASE` for symmetry with https://github.com/openai/openai-python/blob/041bf5a8ec54da19aad0169671793c2078bd6173/openai/__init__.py#LL49C30-L49C30